latex/script/answers.py

```diff
@@ -35,3 +45,28 @@
 )
 ax.set_xlim(-1, 2)
 fig.savefig('build/figures/experience.pdf')
+
+list = []
```

Comment:
You could use a `collections.counter` here instead of looping and creating a pandas series.